mods/fire/init.lua

```diff
@@ -41,11 +42,7 @@ minetest.register_node("fire:basic_flame", {
 
 	on_construct = function(pos)
 		minetest.get_node_timer(pos):start(math.random(30, 60))
-		minetest.after(0, fire.update_sounds_around, pos)
-	end,
-
-	on_destruct = function(pos)
-		minetest.after(0, fire.update_sounds_around, pos)
+		minetest.after(0, minetest.sound_play, "fire_small", {pos = pos, gain = 0.2})
```

Comment:
the `on_destruct` is removed entirely - this code is in the on_construct handler instead.